api.md

```diff
@@ -1874,6 +1877,83 @@ are meant to implement data retention or perform tasks that will improve query
 performance on older chunks. Each policy is assigned a scheduled job
 which will be run in the background to enforce it.
 
+## add_job() :community_function: [](add_job)
+
+Register a custom procedure or function to be scheduled by our automation framework.
+
+#### Required Arguments [](add_job-required-arguments)
+
+|Name|Description|
+|---|---|
+| `proc` | (REGPROC) Name of the function or procedure to register as job|
+| `schedule_interval` | (INTERVAL) Interval between executions of this job|
+
+#### Optional Arguments [](add_job-optional-arguments)
+
+|Name|Description|
+|---|---|
+| `config` | (JSONB) Job-specific configuration (this will be passed to the function when executed)|
+| `initial_start` | (TIMESTAMPTZ) Time of first execution of job. |
+| `scheduled` | (BOOLEAN) Set to `FALSE` to exclude this job from scheduling. Defaults to `TRUE`. |
+
+#### Returns [](add_job-returns)
+
+|Column|Description|
+|---|---|
+|`job_id`| (INTEGER)  TimescaleDB background job id |
+
+
+#### Sample Usage [](add_job-examples)
+
+
+```sql
+SELECT add_job('custom_proc','1h', config:='{"type":"procedure"}'::jsonb);
+```
+
+Register the procedure custom_proc to be run every hour.
+
+## delete_job() :community_function: [](delete_job)
+
+Delete a job registered with the automation framework.
+This works for custom jobs as well as policies.
+
+If the job is currently running the process will be terminated.
+
+#### Required Arguments [](delete_job-required-arguments)
+
+|Name|Description|
+|---|---|
+|`job_id`| (INTEGER)  TimescaleDB background job id |
+
+#### Sample Usage [](delete_job-examples)
+
+
+```sql
+SELECT delete_job(1000);
+```
+
+Delete the job with the job id 1000.
+
+## run_job() :community_function: [](run_job)
+
+Run a previously registered job in the current session.
+This works for custom jobs as well as policies.
+
+#### Required Arguments [](run_job-required-arguments)
+
+|Name|Description|
+|---|---|
+|`job_id`| (INTEGER)  TimescaleDB background job id |
+
+#### Sample Usage [](run_job-examples)
+
+
+```sql
+CALL run_job(1000);
```

Comment:
would be good to give a bit of context and elaborate a bit.
1. select * from timescaledb_information.xxxx and choose a specific policy , e.g.  drop_chunks or refresh policy.
2. then CALL run_job( <job_id>);

Reply:
Clarify use of `CALL` (and perhaps one line why -- this is something new that didn't appear in 1.x).